packages/pnpm-plugin/pnpmfile.cjs

```diff
@@ -0,0 +1,33 @@
+// @ts-check
+module.exports = {
+  hooks: {
+    /** @param {import("@pnpm/types").PnpmSettings & {
+     * publicHoistPattern?: string[],
+     * ignorePatchFailures?: boolean,
+     * packageManagerStrictVersion?: boolean,
+     * minimumReleaseAge?: number,
+     * minimumReleaseAgeExclude?: string[],
+     *
+    }} config */
+    updateConfig(config) {
+      config.publicHoistPattern ??= [
+        '@types*',
+        'eslint',
+        'eslint-config-skuba',
+        'prettier',
+        'esbuild',
+        'jest',
+        'tsconfig-seek',
+        'typescript',
+      ];
```

Comment:
To support merging, the array configs could be like

```suggestion
      (config.publicHoistPattern ??= []).unshift(
        '@types*',
        'eslint',
        'eslint-config-skuba',
        'prettier',
        'esbuild',
        'jest',
        'tsconfig-seek',
        'typescript',
      );
```

Or, if the above is too spooky to look at:

```suggestion
      config.publicHoistPattern = [
        '@types*',
        'eslint',
        'eslint-config-skuba',
        'prettier',
        'esbuild',
        'jest',
        'tsconfig-seek',
        'typescript',
        ...(config.publicHoistPattern ?? []),
      ];
```

Reply:
[This](https://github.com/SEEK-Jobs/indie-awsk-ui/pull/349) seems to imply that plugins don't work properly with `pnpm fetch` and `pnpm install --offline` 🤔